### PR TITLE
Feat/offline

### PR DESCRIPTION
**Pull request by JeanHaderer**

### CODE REVIEW

File: src/orchestrator/Orchestrator.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    const { source, data } = props;
+    const { goPreviousPage, goNextPage, isLastPage, isFirstPage, getComponents, getCurrentErrors } =
+        lunatic.useLunatic(source, data, {
+            onChange: onLogChange,
+        });
+    const components = getComponents();
+    const currentErrors = getCurrentErrors();
+
+    return (
+        <ThemeProvider>
```
Comment (ddecrulle) [inline src/orchestrator/Orchestrator.tsx:44]: Orchestrator is Already Scoped into <ThemeProvider>

File: src/App.tsx [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+
+        fetch(url)
+            .then(sourcePromise => sourcePromise.json())
+            .then(source => setSource(source));
+    }, []);
+
+    return source && data ? (
+        <ThemeProvider>
+            <CssBaseline enableColorScheme />
+            <Home></Home>
```
Comment (ddecrulle) [inline src/App.tsx:27]: If there is no children, please use self-closing tag

File: config-overrides.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const path = require("path");
+
+module.exports = function override(config, env) {
+    config.resolve.alias["react"] = path.resolve('./node_modules/react');
+    config.resolve.alias["react-dom"] = path.resolve('./node_modules/react-dom');
+    console.log(config);
```
Comment (ddecrulle) [inline config-overrides.js:6]: can you delete this console.log please

File: src/App.tsx [hunk 4]
```diff
@@ -1,26 +1,35 @@
...
-        </div>
+import { CssBaseline } from "@mui/material";
+import { OrchestratorForStories } from "./orchestrator/Orchestrator";
+import { ThemeProvider } from "lunatic-edt";
+import Home from "./page/home/Home";
+import "i18n/i18n";
+
+const App = () => {
+    const data = {};
+    const [source, setSource] = useState(null as object | null);
```
Comment (ddecrulle) [inline src/App.tsx:12]: I'm not very convince by `null as object | null`

**Review by ddecrulle — CHANGES_REQUESTED** [review]

File: src/orchestrator/Orchestrator.tsx (hunk 1 above)
Comment (JeanHaderer) [reply src/orchestrator/Orchestrator.tsx:44]: This on I don't understand

Comment (ddecrulle) [reply src/orchestrator/Orchestrator.tsx:44]: if you look [here](https://github.com/InseeFrLab/edt/blob/feat/offline/src/App.tsx#L25-L29) you see that Orchestrator is inside <ThemeProvider> so should not rewrap by <ThemeProvider> in this file

File: src/App.tsx (hunk 4 above)
Comment (JeanHaderer) [reply src/App.tsx:12]: J'ai essayé de créer le type source pour lunatic mais c'était très complexe. Il faut vraiment une très bonne connaissance du modèle. Laissons object | null pour le moment.

**Pull request merged**